### PR TITLE
Changes made to make scripts better

### PR DESCRIPTION
**Pull request by meghasemim1999**

### CODE REVIEW

File: Deployment/Kubernetes/install-kubeadm.sh [hunk 1]
```diff
@@ -31,11 +31,10 @@ then
 	sudo apt-get install -y apt-transport-https ca-certificates curl
 
 	# Download the Google Cloud public signing key:
-	sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
-
+	sudo curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
```
Comment (soroshsabz) [inline Deployment/Kubernetes/install-kubeadm.sh]: why remove flags of curl?

File: Deployment/Kubernetes/install-kubeadm.sh [hunk 2]
```diff
@@ -31,11 +31,10 @@ then
...
 	# Add the Kubernetes apt repository:
 	# TODO: xenial must based on edition
-	echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] http://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
```
Comment (soroshsabz) [inline Deployment/Kubernetes/install-kubeadm.sh]: I do not know, why this change is useful?

File: Deployment/Kubernetes/install-kubeadm.sh [hunk 3]
```diff
@@ -31,11 +31,10 @@ then
 	sudo apt-get install -y apt-transport-https ca-certificates curl
 
 	# Download the Google Cloud public signing key:
-	sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
```
Comment (soroshsabz) [inline Deployment/Kubernetes/install-kubeadm.sh]: I suggest, in first stage download gpg key, and then add this key, if you want, with sudo `apt-key add`

**Review by soroshsabz — CHANGES_REQUESTED** [review]

File: Deployment/Kubernetes/install-kubeadm.sh (hunk 3 above)
Comment (soroshsabz) [reply Deployment/Kubernetes/install-kubeadm.sh]: Why does not save this key in /usr/share/keyrings/kubernetes-archive-keyring.gpg ?

File: Deployment/Kubernetes/install-kubeadm.sh [hunk 4]
```diff
@@ -31,11 +31,14 @@ then
...
+	sudo curl -fsSLo https://packages.cloud.google.com/apt/doc/apt-key.gpg 
+	
+	# Add key
+	sudo apt-key add apt-key.gpg
```
Comment (soroshsabz) [inline Deployment/Kubernetes/install-kubeadm.sh]: why add system wide?

File: Deployment/Kubernetes/install-kubeadm.sh (hunk 2 above)
Comment (soroshsabz) [reply Deployment/Kubernetes/install-kubeadm.sh]: please response it

**Review by soroshsabz — CHANGES_REQUESTED** [review]

**Comment by meghasemim1999** [comment]
all unneccessary changes are removed.

@soroshsabz

**Review by soroshsabz — APPROVED** [review]

**Pull request merged**